resolwe/flow/tests/test_executors.py

```diff
@@ -267,6 +267,9 @@ def test_run_in_docker(self):
         data = self.run_process("test-docker")
         self.assertEqual(data.output["result"], "OK")
 
+    @unittest.skipIf(
```

Comment:
This is not needed, as you remove it later anyway.

Reply:
Done.